src/resources/js/events-admin.js

```diff
@@ -373,21 +373,26 @@ jQuery( document ).ready( function( $ ) {
 
 	var toggle_linked_post_fields = function( event ) {
 
-		var $select    = $( this );
-		var selectData = $select.data( 'select2' );
-		var $group     = $select.closest( 'tbody' );
-		var $edit      = $group.find( '.edit-linked-post-link a' );
-		var value      = $select.val();
-		var editLink   = '';
-
-		if ( $select.find( ':selected' ).val() == value ) {
-			editLink = $select.find( ':selected' ).data( 'editLink' );
+		const $select = $( this );
+		const $group = $select.closest( 'tbody' );
+		const $edit = $group.find( '.edit-linked-post-link a' );
+		const value = $select.val();
+		const $selected = $select.find( ':selected' );
+		const selectedVal = $selected.val();
+		let editLink = '';
+		let existingPost = false;
+
+		if ( selectedVal === value ) {
+			editLink = $selected.data( 'editLink' );
+			existingPost = !! $selected.data( 'existingPost' );
 		}
 
+		$edit.hide();
```

Comment:
Without this here, the edit link would keep showing after you've selected a venue and go to create a new venue.

Reply:
Lets add a comment about that?